HybridWebControl/Helpers/UserAgentParser.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HybridWebControl
+{
+    public static class UserAgentParser
+    {
+        public static string ConvertToNativeBrowser(this String userAgent)
```

Comment:
I suggest ConvertUserAgentToNativeBrowser which is long although, but explicit and clear, then safe